iocage/lib/ioc_plugin.py

```diff
@@ -477,11 +477,9 @@ def __fetch_plugin_install_packages__(self, uuid, jaildir, conf, _conf,
         if err:
             iocage.lib.ioc_common.logit(
                 {
-                    "level":
-                    "EXCEPTION",
-                    "message":
-                    "pkg error, refusing to fetch artifact and "
-                    "run post_install.sh!\n"
+                    "level": "EXCEPTION",
+                    "message": f"\npkg error:\n  - {err}\n"
+                    "\nRefusing to fetch artifact and run post_install.sh!"
```

Comment:
Is it because I am not native speaker that Refusing looks very strange in that sentence?

Wouldn't "Failed" make more sense in that sentence?